client/state/analytics/actions.js

```diff
@@ -18,13 +24,16 @@ const mergedMetaData = ( a, b ) => [
 
 const joinAnalytics = ( analytics, action ) =>
 	isFunction( action )
-		? dispatch => { dispatch( analytics ); dispatch( action ); }
+		? dispatch => {
+			dispatch( analytics );
+			dispatch( action );
+		}
```

Comment:
hrm. was this a style guide issue that led to this change?

I'm not a super fan of how I wrote it originally, but I also don't like the way the `}` lines up and how it throws off the ternary indentation

(not a blocker comment)